app/assets/javascripts/short-form/ShortFormHelperService.js.coffee

```diff
@@ -83,12 +83,19 @@ ShortFormHelperService = ($translate, $filter, $sce, $state) ->
       markup = "#{nameEditable} <a class='lined' href='#{link}'>#{accountSettings}</a>"
     return $sce.trustAsHtml(markup)
 
-  Service.addressTranslationVariable = (address) ->
+  Service.addressTranslateVariable = (address) ->
     { address: address }
 
-  Service.membersTranslationVariable = (members) ->
+  Service.membersTranslateVariable = (members) ->
     { user: $filter('listify')(members) }
 
+  Service.youOrHouseholdTranslateVariable = (membersCount, wholeHousehold) ->
+    value = if membersCount > 0
+      if wholeHousehold then 'does your household' else 'you or anyone on this application'
```

Comment:
@kaptron Good call! I like the approach of removing the do/does and having a an ng-if='' depending if it's singular or plural. Can we create this as a chore for the next sprint since we've already set a goal for this sprint?